### PR TITLE
Add Shrine normalizer

### PR DESCRIPTION
**Pull request by janko**
[Shrine](https://github.com/shrinerb/shrine) has added a plugin for [instrumentation](https://github.com/shrinerb/shrine#logging) in version 2.19.0, which goes via ActiveSupport::Notifications. So I was thinking that it would be nice to add a normalizer for it in Skylight, akin to the ActiveStorage one.

I only had problems with testing it, as I had troubles activating Skylight in a non-Rails environment. I've created an app (`DZJxlbrBTeYt`), and tried running the following script:

```
$ gem install skylight shrine shrine-memory
```
```rb
ENV["SKYLIGHT_AUTHENTICATION"] = "<MY_SECRET>"

require "skylight"
require "active_support/all"
require "shrine"
require "shrine/storage/memory"

Skylight.start!
Skylight.enable_normalizer("shrine")

Shrine.storages[:memory] = Shrine::Storage::Memory.new

Shrine.plugin :instrumentation

uploaded_file = Shrine.upload(StringIO.new, :memory)
uploaded_file.exists?
uploaded_file.download
uploaded_file.delete
```

But I don't see any events in my Skylight dashboard, even though Shrine did send out the events via `ActiveSupport::Notifications`.

### CODE REVIEW

**Comment by wagenet** [comment]
@cla-bot check

**Comment by wagenet** [comment]
@janko we really dropped the ball on this one. What needs to be done at this point to get this merged?

**Comment by janko** [comment]
Everything should still be working with newest version of Shrine, so if nothing has changed on Skylight side I say that we can merge this.

File: skylight-core/lib/skylight/core/normalizers/shrine.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "data_uri.shrine"         => "Data URI",
+        "remote_url.shrine"       => "Remote URL",
+      }.freeze
+
+      TITLES.each_key do |key|
+        register key
+      end
+
+      def normalize(_trace, name, _payload)
+        puts "NORMALIZE"
```
Comment (wagenet) [inline skylight-core/lib/skylight/core/normalizers/shrine.rb]: Can you remove this puts?

Comment (janko) [reply skylight-core/lib/skylight/core/normalizers/shrine.rb]: Oops, removed

**Comment by janko** [comment]
I'm not sure what kind of endpoint do you mean. What I tried to do was run a Ruby script that loads the Skylight agent and runs some Shrine operations that triggers ActiveSupport notifications, and then check in my Skylight dashboard whether I see these events. However, as I've mentioned, I didn't see any data in my dashboard, as if though I didn't set up the Skylight agent properly.

Perhaps this all needs to be part of a web app in order for data to show up? I could create a simple Sinatra app which uses Shrine and put it on Heroku. I don't know if that's what you meant, I'm not sure how helpful it would be to you.

**Comment by wagenet** [comment]
@janko it doesn't necessarily need to be in a web app. However, it's definitely possible you're missing a bootstrapping step. Can you share your test scripts so I can give them a whirl?

**Comment by janko** [comment]
Yes, it's posted in the PR description.

**Comment by wagenet** [comment]
@janko of course, I'm sorry! I'll try to give this some more review today or early next week.

**Comment by wagenet** [comment]
@janko I'm so sorry this took so long to merge. Thanks again!

**Pull request merged**